widgets/thumbs/thumbs.css

```diff
@@ -61,32 +62,37 @@
   height: 28px;
   font-weight: bold;
   text-align: center;
-  background: url(thumbs.png) 0 0 no-repeat;
+  background: url('thumbs.png') 0 0 no-repeat;
```

Comment:
No need for that change, see [CSS coding standards for properties](https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines#properties).
Please do this change for all the instances.